### PR TITLE
Add validator registration buffer

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/commands/start.py [hunk 1]
```diff
@@ -207,6 +207,13 @@
     envvar='POOL_SIZE',
     type=int,
 )
+@click.option(
+    '--min-validators-required',
```
Comment (tsudmi) [inline src/commands/start.py:211]:
```suggestion
    '--min-validators-registration',
```

File: src/config/settings.py [hunk 2]
```diff
@@ -20,6 +20,8 @@
 DEFAULT_API_HOST = '127.0.0.1'
 DEFAULT_API_PORT = 8000
 
+DEFAULT_VALIDATORS_REQUIRED = 1
```
Comment (evgeny-stakewise) [inline src/config/settings.py:23]: Must be 'min': `DEFAULT_MIN_VALIDATORS_...` not sure about suffix

File: src/validators/tasks.py [hunk 3]
```diff
@@ -127,6 +127,15 @@ async def register_validators(
         # not enough balance to register validators
         return None
 
+    # Check if there is enough ETH to register the specified minimum number of validators
+    if validators_count < settings.min_validators_required:
+        logger.info(
```
Comment (evgeny-stakewise) [inline src/validators/tasks.py:132]: Will be spamming on each block. Do we need it? I would like to decrease logging amount somehow.

Comment (evgeny-stakewise) [reply src/validators/tasks.py:132]: maybe log only in verbose mode

File: src/commands/start.py [hunk 4]
```diff
@@ -207,6 +207,13 @@
     envvar='POOL_SIZE',
     type=int,
 )
+@click.option(
+    '--min-validators-registration',
+    type=int,
+    envvar='MIN_VALIDATORS_REQUIRED',
```
Comment (cyc60) [inline src/commands/start.py:213]: should be `min_validators_registration`?

**Review by evgeny-stakewise — APPROVED** [review]

**Review by cyc60 — APPROVED** [review]

**Pull request merged**